### ISSUE TITLE
`paraguay_dncp_*` get data by month

### ISSUE THREAD

**Issue by yolile** [open-contracting/kingfisher-collect#882]
Currently, the pagination fails for pages larger than 1000. We can still get the data without errors if we always use enough filters, for example, requesting one month at a time.

**Comment by yolile** [comment]
The API is still returning more than 10000 results for some months, so we need to reduce the interval to, for example, 15 days

**Comment by Ravf95** [comment]
Hi @yolile
I found  [here](https://contrataciones.gov.py/datos/api/v3/doc/search/processes?tipo_fecha=fecha_release&fecha_desde=2019-08-16T20%3A17%3A11-04%3A00&fecha_hasta=2019-08-31T20%3A17%3A11-04%3A00&items_per_page=10000&page=1) an exception from the current interval (15 days), so I also checked with another interval (7 days) and got [this](https://contrataciones.gov.py/datos/api/v3/doc/search/processes?tipo_fecha=fecha_release&fecha_desde=2013-03-03T23:52:18-04:00&fecha_hasta=2013-03-10T23:52:18-04:00&items_per_page=10000 ) exception. At [2013-03-05T08:24:59-04:00](https://contrataciones.gov.py/datos/api/v3/doc/search/processes?tipo_fecha=fecha_release&fecha_desde=2013-03-05T08:24:59-04:00&fecha_hasta=2013-03-05T08:24:59-04:00&items_per_page=10000) I found 22448 results, we can't scrape that.

I think we can make the interval change dynamically (1, 7, 15, 30 days) if `pagination/total_items` exceeds 10000 items or maybe set a default value and change it with an environment variable. What do you think?

**Comment by yolile** [comment]
Thanks for checking @Ravf95. Right now we are using the "fecha_release" date type, but another option is to use the "publicacion_llamado" date type, which is the date of when the tender was published. I checked one of the collections that we have from Paraguay in kingfisher and there are no more than 3k published tenders by month, so we can use 30 days and that date type. However, for tenders without a publication date, for example, the ones that never passed the planning stage, we need to use another URL. We can use the filter tender.id = 'planned' (I found more than 40k plannings in kingfisher) + the release_date filter.
I've made that change in #902
